deploy/kustomize/overlays/cloudbeat-eks/cloudbeat.yml

```diff
@@ -2,9 +2,11 @@ cloudbeat:
   config:
     v1:
       benchmark: cis_eks
+      aws:
+        credentials:
+          access_key_id: ${AWS_ACCESS_KEY_ID:no_access_key_was_set}
+          secret_access_key: ${AWS_SECRET_ACCESS_KEY:no_secret_was_set}
```

Comment:
@uri-weisman, the file `tests/deploy/cloudbeat-eks-pytest.yaml` should be also updated with the code above.

Reply:
@gurevichdmitry - thanks updated.